src/main/java/com/sngular/kloadgen/serializer/EnrichedRecordSerializer.java

```diff
@@ -0,0 +1,7 @@
+package com.sngular.kloadgen.serializer;
+
+import org.apache.kafka.common.serialization.Serializer;
+
+public interface EnrichedRecordSerializer<T extends EnrichedRecord> extends Serializer<T> {
```

Comment:
There is no sense on that. Rethink.

Reply:
This is a tag for those serializers that can accept a EnricherdRecord, instead of a GenericRecordSerializer. Is there a way to query this programatically?